vitest.extend.ts

```diff
@@ -49,17 +60,15 @@ chai.util.addMethod(
     }: ObjectOptions = {},
     hint?: string,
   ) {
-    let snap: Record<string, unknown>;
     const received = chai.util.flag(this, 'object');
+    let snap: Record<string, unknown> = received;
 
-    if (received instanceof FabricObject) {
```

Comment:
this was something that I don't know how it even worked
we can't import fabric objects as values in setup file
because it can break mocks in tests
this failed in browser mode because in one test we mocked uid module, but fabric object extends soemthign that imports uid and then mock of the uid module failed in the test file

so I opted to not use the import as value but to just use duck typed check which worked fine

Reply:
adjusted as per other comment